bfcfg

```diff
@@ -648,22 +655,42 @@ sys_cfg_one_byte()
   fi
 }
 
+sys_cfg_multi_byte()
```

Comment:
Could you please leave a comment to indicate that this function only displays the value? It doesn't set it (unlike sys_cfg_one_byte)

I would also keep the number of arguments the same as sys_cfg_one_byte, although the value won't be ignored. Who knows maybe we would need to configure other multi bytes sys config in the future

Reply:
Hey Khalil, I updated the function name and added a comment to make it more clear that this is for dumping.

I left out support for setting multi-byte values because it isn't needed yet and we have to be more careful about correctly handling the input size of the arguments passed to bfcfg when setting variables to avoid overwriting memory in the `SYS_CONFIG` struct. For that reason I also removed the "value" input argument.

In the future it probably makes sense to remove the distinction of `sys_cfg_one_byte` and `sys_cfg_multi_byte` and combine into one common function that can set and dump any number of bytes at a given offset. Didn't want to do that here because it would increase the regression test load for a case that isn't needed yet, but happy to do in the future if we want to add that support.